docs/guides/tutorials/nextjs.rst

```diff
@@ -0,0 +1,529 @@
+
+Open `localhost:3000 <http://localhost:3000>`_ to see the default Next.js homepage. At this point the app's file structure looks like this:
+
+.. code-block::
+
+  README.md
+  tsconfig.json
+  package.json
+  next.config.js
+  next-env.d.ts
+  pages
+  ├── _app.tsx
+  ├── api
+  │   └── hello.ts
+  └── index.tsx
+  public
+  ├── favicon.ico
+  └── vercel.svg
+  styles
+  ├── Home.module.css
+  └── globals.css
+
+There's a custom App component defined in ``pages/_app.tsx`` that loads some global CSS, plus the homepage at ``pages/index.tsx`` and a single API route at ``pages/api/hello.ts``. The ``styles`` and ``public`` directories contain some other assets.
+
+Updating the homepage
+---------------------
+
+Let's start by implementing a simple homepage for our blog application using static data. Replace the contents of ``pages/index.tsx`` with the following.
+
+.. code-block:: typescript
+
+  // pages/index.tsx
+
+  import type {NextPage} from 'next';
+  import Head from 'next/head';
+  import styles from '../styles/Home.module.css';
+
+  type Post = {
+    id: string;
+    title: string;
+    content: string;
+  };
+
+  const HomePage: NextPage = () => {
+    const posts: Post[] = [
+      {
+        id: 'post1',
+        title: 'This one weird trick makes using databases fun',
+        content: 'Use EdgeDB',
+      },
+      {
+        id: 'post2',
+        title: 'How to build a blog with EdgeDB and Next.js',
+        content: "Let's start by scaffolding out app with `create-next-app`.",
+      },
+    ];
+
+    return (
+      <div className={styles.container}>
+        <Head>
+          <title>My Blog</title>
+          <meta name="description" content="An awesome blog" />
+          <link rel="icon" href="/favicon.ico" />
+        </Head>
+
+        <main className={styles.main}>
+          <h1 className={styles.title}>Blog</h1>
+          <div style={{height: '50px'}}></div>
+          {posts.map((post) => {
+            return (
+              <a href={`/post/${post.id}`} key={post.id}>
+                <div className={styles.card}>
+                  <p>{post.title}</p>
+                </div>
+              </a>
+            );
+          })}
+        </main>
+      </div>
+    );
+  };
+
+  export default Home;
+
+After saving, Next.js should hot-reload, and the homepage should look something like this.
+
+
+.. image::
+    http://localhost:3000/docs/tutorials/nextjs/basic_home.png
+    :alt: Basic blog homepage with static content
+    :width: 100%
+
+Initializing EdgeDB
+-------------------
+
+Now let's spin up a database for the app. First, install the ``edgedb`` CLI.
+
+**Linux or macOS**
+
+.. code-block:: bash
+
+    $ curl --proto '=https' --tlsv1.2 -sSf https://sh.edgedb.com | sh
+
+**Windows Powershell**
+
+.. code-block:: powershell
+
+    PS> iwr https://ps1.edgedb.com -useb | iex
+
+Once the CLI is installed, initialize a project from the application's root directory. You'll be presented with a series of prompts.
```

Comment:
I'd mention that the user is supposed to read what the command prints. Potentially the terminal will have to be reopen in order for `edgedb` command to become available.. 

(because the installation script alters `.bashrc` to add itself to the PATH)